src/Data/Text/Internal/Builder.hs

```diff
@@ -181,6 +186,12 @@ fromText t@(Text arr off l)
 --
 --  * @'toLazyText' ('fromString' s) = 'L.fromChunks' [S.pack s]@
 --
+-- Performs replacement on invalid scalar values:
+--
+-- >>> fromString "\55555"
+-- "\65533"
+--
+-- @since 1.2.0.0
```

Comment:
Are you sure? It seems available earlier.

Reply:
That's inadvertent from a copy-paste rollback. It'll be removed.